src/app/cake/cake.model.ts

```diff
@@ -0,0 +1,10 @@
+export type Status = 'Available' | 'LastUnits' | 'OutOfStock';
```

Comment:
This should be an Enumerator.

Since we are using a Single File Responsibility file distribution, this Enumerator should live in a separate file named `cake-status.enum.ts`.